### PR TITLE
SONAR-7516 Change capitalization of metric names

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 1]
```diff
@@ -2248,15 +2248,15 @@ metric.generated_lines.name.suffix=generated lines
 metric.generated_lines.description=Number of generated lines
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Number of Generated Lines

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 2]
```diff
@@ -2291,13 +2291,13 @@ metric.file_complexity.name=Complexity /file
 metric.file_complexity.description=Complexity average by file
 metric.file_complexity.abbreviation=Cmpx/file
 
-metric.class_complexity_distribution.name=Classes distribution /complexity
+metric.class_complexity_distribution.name=Classes Distribution /complexity
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Classes Distribution / Complexity

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 3]
```diff
@@ -2291,13 +2291,13 @@ metric.file_complexity.name=Complexity /file
...
 metric.class_complexity_distribution.description=Classes distribution /complexity
 
-metric.function_complexity_distribution.name=Functions distribution /complexity
+metric.function_complexity_distribution.name=Functions Distribution /complexity
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Functions Distribution / Complexity

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 4]
```diff
@@ -2291,13 +2291,13 @@ metric.file_complexity.name=Complexity /file
...
 metric.function_complexity_distribution.description=Functions distribution /complexity
 
-metric.file_complexity_distribution.name=Files distribution /complexity
+metric.file_complexity_distribution.name=Files Distribution /complexity
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: File Distribution / Complexity

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 5]
```diff
@@ -2307,100 +2307,100 @@ metric.file_complexity_distribution.description=Files distribution /complexity
 #
 #--------------------------------------------------------------------------------------------------------------------
 
-metric.tests.name=Unit tests
+metric.tests.name=Unit Tests
 metric.tests.description=Number of unit tests
 metric.tests.abbreviation=UTs
 
-metric.test_execution_time.name=Unit tests duration
+metric.test_execution_time.name=Unit Tests Duration
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Unit Test Duration

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 6]
```diff
@@ -2307,100 +2307,100 @@ metric.file_complexity_distribution.description=Files distribution /complexity
...
 metric.test_execution_time.short_name=Duration
 metric.test_execution_time.description=Execution duration of unit tests
 metric.test_execution_time.abbreviation=UTs dur.
 
-metric.test_errors.name=Unit tests errors
+metric.test_errors.name=Unit Tests Errors
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Unit Test Errors

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 7]
```diff
@@ -2307,100 +2307,100 @@ metric.file_complexity_distribution.description=Files distribution /complexity
...
 metric.test_errors.abbreviation=UTs errors
 
-metric.skipped_tests.name=Skipped unit tests
+metric.skipped_tests.name=Skipped Unit Tests
 metric.skipped_tests.short_name=Skipped
 metric.skipped_tests.description=Number of skipped unit tests
 metric.skipped_tests.abbreviation=Skipped UTs
 
-metric.test_failures.name=Unit tests failures
+metric.test_failures.name=Unit Tests Failures
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Unit Test Failures

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 8]
```diff
@@ -2307,100 +2307,100 @@ metric.file_complexity_distribution.description=Files distribution /complexity
...
 metric.test_failures.short_name=Failures
 metric.test_failures.description=Number of unit test failures
 metric.test_failures.abbreviation=UTs failures
 
-metric.test_success_density.name=Unit tests success (%)
+metric.test_success_density.name=Unit Tests Success (%)
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Unit Test Success

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 9]
```diff
@@ -2307,100 +2307,100 @@ metric.file_complexity_distribution.description=Files distribution /complexity
...
 metric.test_success_density.short_name=Success
 metric.test_success_density.description=Density of successful unit tests
 metric.test_success_density.abbreviation=UTs success
 
-metric.test_data.name=Unit tests details
+metric.test_data.name=Unit Tests Details
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Unit Test Details

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 10]
```diff
@@ -2525,24 +2525,24 @@ metric.new_overall_branch_coverage.description=Condition coverage of new/changed
...
 metric.duplicated_files.abbreviation=Dup. files
 
-metric.duplicated_lines_density.name=Duplicated lines (%)
+metric.duplicated_lines_density.name=Duplicated Lines (%)
 metric.duplicated_lines_density.short_name=Duplications
 metric.duplicated_lines_density.description=Duplicated lines balanced by statements
 metric.duplicated_lines_density.abbreviation=Dup. lines(%)
 
-metric.duplications_data.name=Duplications details
+metric.duplications_data.name=Duplications Details
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Duplication Details

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 11]
```diff
@@ -2645,77 +2645,77 @@ metric.noc.description=Number of Children
 metric.rfc.name=Response for Class
 metric.rfc.description=Response for Class
 
-metric.rfc_distribution.name=Class distribution /RFC
+metric.rfc_distribution.name=Class Distribution /RFC
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Distribution / RFC

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 12]
```diff
@@ -2645,77 +2645,77 @@ metric.noc.description=Number of Children
...
 metric.rfc_distribution.description=Class distribution /RFC
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Class distribution / RFC

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 13]
```diff
@@ -2645,77 +2645,77 @@ metric.noc.description=Number of Children
...
 
 metric.lcom4.name=LCOM4
 metric.lcom4.description=Lack of Cohesion of Functions
 
-metric.lcom4_blocks.name=LCOM4 blocks
+metric.lcom4_blocks.name=LCOM4 Blocks
 metric.lcom4_blocks.description=LCOM4 blocks
 
-metric.lcom4_distribution.name=Class distribution /LCOM4
+metric.lcom4_distribution.name=Class Distribution /LCOM4
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Class Distribution / LCOM4

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties (hunk 1 above)
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: metric.commented_out_code_lines.name=Commented-Out LOC
metric.class_complexity.name=Complexity / Class
metric.function_complexity.name=Complexity / Function
metric.file_complexity.name=Complexity / File

File: sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java [hunk 14]
```diff
@@ -429,7 +429,7 @@
       .create();
 
   public static final String FILE_COMPLEXITY_DISTRIBUTION_KEY = "file_complexity_distribution";
-  public static final Metric<String> FILE_COMPLEXITY_DISTRIBUTION = new Metric.Builder(FILE_COMPLEXITY_DISTRIBUTION_KEY, "Files distribution /complexity", Metric.ValueType.DISTRIB)
+  public static final Metric<String> FILE_COMPLEXITY_DISTRIBUTION = new Metric.Builder(FILE_COMPLEXITY_DISTRIBUTION_KEY, "Files Distribution / Complexity", Metric.ValueType.DISTRIB)
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java]: MINOR Split this 181 characters long line (which is greater than 180 authorized). [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00103)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 15]
```diff
@@ -2645,77 +2645,77 @@ metric.noc.description=Number of Children
 metric.rfc.name=Response for Class
 metric.rfc.description=Response for Class
 
-metric.rfc_distribution.name=Class distribution /RFC
+metric.rfc_distribution.name=Class distribution / RFC
```
Comment (ganncamp) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: Class Distribution / RFC

**Pull request merged**